.github/policies/eventResponder.yml

```diff
@@ -282,3 +282,17 @@ configuration:
               reply: |
                 > [!NOTE]
                 > The "Status: In PR :point_right:" label was removed as per [ITA23](https://azure.github.io/Azure-Verified-Modules/help-support/issue-triage/issue-triage-automation/#ita23).
+
+      - description: 'ITA25 - When #closethis is used in an issue, close the issue.'
```

Comment:
Question: Does that enable any person with a GitHub account to close our issues?

Reply:
As it stands, yes, which made me realize that we said this should be filtered to the "hygiene" issues that triggered the need for this feature. I'll add the filtering shortly. Thanks for calling it out!

Reply:
Filter criteria updated.

Reply:
I would have the same concern. Everyone can comment on an issue while only specific people should be able to close it. If I wore a black hat I'd probably start commenting randomly, but maybe I'm missing something

Reply:
@eriqua, I shared more context for this one in the core team's internal chat.